ansible/roles/collectory/tasks/main.yml

```diff
@@ -7,8 +7,30 @@
     - deploy
     - config
 
+
+- name: Set service user  - tomcat or a systemd service user (collectory by default)
+  set_fact:
+    data_owner_user: "{{ collectory_service_system_user if exec_jar else tomcat_user }}"
+
+- name: run collectory on Tomcat or as systemd
+  debug:
+    msg: >
+      {% if exec_jar | default(false) | bool %}
+        Running as a systemd service.
+      {% else %}
+        Running on {{tomcat}}.
+      {% endif %}
+      Version: {{ version }}.{{ packaging }}
+      Service user: {{ data_owner_user }}
+
+- name: "ensure system user for collectory exists - "
```

Comment:
Add a conditional to run this `ensure system user for collectory exists` so it is not run for a tomcat deploy.

Reply:
It sets on line 11: data_owner_user: "{{ collectory_service_system_user if exec_jar else tomcat_user }}"

Reply:
The reason for the conditional is because I do not trust ansible to create the tomcat user in an identical fashion to the tomcat package installer. While the tomcat user creation should take place before this task it is not certain.

Reply:
Done